package.json

```diff
@@ -46,13 +46,13 @@
     "heroku-postbuild": "yarn build:esm && yarn --cwd demo install && yarn --cwd demo build",
     "demo": "yarn && yarn build:esm && yarn build:demo && yarn start",
     "build:test:functional": "yarn && yarn build:esm && yarn --cwd demo  test:functional",
-    "test:functional":"yarn --cwd demo  test:functional"
+    "test:functional": "yarn --cwd demo  test:functional"
```

Comment:
@johnbenjamin-hmcts - might as well take out the extra space between `demo` and `test` :)

Reply:
Thank you @Daniel-Lam, running the command `yarn install` auto-corrected the spacing after `:` but not the other one :)